plugins/modules/zos_job_submit.py

```diff
@@ -718,6 +718,28 @@ def run_module():
         else:
             # For local file, it has been copied to the temp directory in action plugin.
             from_encoding = encoding.get("from")
+
+            # This pre-filter from 8859-1 removes c/r (\r 0x0d) because iconv can't
+            if from_encoding == "ISO8859-1":
```

Comment:
It could make sense from type=USS, but I thought (assumed) that if a file is in a data set, it should already be in EBCDIC.  Maybe we should add this to USS-origin.  Yes, limiting "from' 8859-1" is is little chancey, but that was the case that kept happening.